gateway/conf/nginx.conf.liquid

```diff
@@ -131,8 +134,10 @@ http {
 
     {% if https_port -%}
     listen {{ https_port }} ssl;
-    ssl_certificate {{ env.APICAST_HTTPS_CERTIFICATE }};
-    ssl_certificate_key {{ env.APICAST_HTTPS_CERTIFICATE_KEY }};
+    ssl_certificate {{ env.APICAST_HTTPS_CERTIFICATE | default: "conf/server.crt" | filesystem | first }};
```

Comment:
Why is `first` needed?

Reply:
because `| filesystem` produces a table and it can't be serialized.